src/api.ts

```diff
@@ -1,23 +1,134 @@
-import { Bytes } from "./utils";
+import { assert, Bytes, NearAmount, PromiseIndex, Register } from "./utils";
 import { PromiseResult } from "./types";
 
 const U64_MAX = 2n ** 64n - 1n;
 const EVICTED_REGISTER = U64_MAX - 1n;
 
 // Interface available in QuickJS
 interface Env {
-  panic_utf8: (msg: string) => never;
-  [x: string]: any;
+  panic_utf8(msg: string): never;
```

Comment:
Nice additions to the env interface!

most of the functions here should take Bytes instead of string, please referring to the implementation function below. For example,
```js
export function panicUtf8(msg: Bytes): never {
  env.panic_utf8(msg);
}
```

so panicUtf8 take a Bytes

Reply:
@ailisp Will update those definitions to take `Bytes` instead.